projects/bp-gallery/src/components/views/geoguessr/GeoMap.tsx

```diff
@@ -34,9 +35,13 @@ const PlayerMarkers = ({
     }, 300);
   }, [coords, myGuess, map]);
   const othersIcon = new Icon({
-    ...Icon.Default.prototype.options,
     iconUrl: otherMarkerIcon,
+    iconSize: Icon.Default.prototype.options.iconSize,
+    iconAnchor: Icon.Default.prototype.options.iconAnchor,
+    popupAnchor: Icon.Default.prototype.options.popupAnchor,
     shadowUrl: markerShadow,
+    shadowSize: Icon.Default.prototype.options.shadowSize,
+    shadowAnchor: Icon.Default.prototype.options.shadowAnchor,
```

Comment:
You could use [`pick`](https://docs-lodash.com/v4/pick/) to avoid the repetition:
```ts
    ...pick(Icon.Default.prototype.options, [
      'iconSize',
      'iconAnchor',
      'popupAnchor',
      'shadowSize',
      'shadowAnchor',
    ]),
    iconUrl: myMarkerIcon,
    shadowUrl: markerShadow,
```